drivers/framework.go

```diff
@@ -24,6 +26,7 @@ func RegisterDrivers() {
 	Drivers["scaleService"] = &ScaleServiceDriver{}
 	Drivers["serviceUpgrade"] = &ServiceUpgradeDriver{}
 	Drivers["scaleHost"] = &ScaleHostDriver{}
+	Drivers["serviceWebhook"] = &ServiceWebhookDriver{}
```

Comment:
I don't think "serviceWebhook" describes what this driver does very well.
What do you think of "forwardRequest"?

Reply:
@cjellick We actually don't forward all kinds of requests, I mean like `get`, `put`, `delete`. So,  maybe `forwardRequest` doesn't describe it well. How about `forwardWebhook`?